services/sources/met_service.py

```diff
@@ -26,7 +26,7 @@ def get_records(
         offset: Optional[int]=None,
         limit: Optional[int]=None
     ) -> Generator[METMapping, None, None]:
-        current_position = offset
+        current_position = offset if offset is not None else 0
```

Comment:
Can the parameter definition be updated to 0 instead of None to eliminate the need for this logic?

Reply:
yes - good callout!